Makefile

```
@@ -152,8 +152,43 @@ install-pipelines: ## Install the CNBI pipelines into the configured cluster.
 uninstall-pipelines: ## Uninstall the CNBI pipelines from  the configured cluster. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	kubectl delete --ignore-not-found=$(ignore-not-found) -f 'hack/cnbi-*.yaml'
 
+define managed_operators
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator
+  namespace: openshift-operators
+spec:
+  channel: latest
```

can we pin this to a specific version? so that we (could if we would) fulfill external requirements for a specific version?

---

https://docs.openshift.com/container-platform/4.11/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-specific-version-cli_olm-adding-operators-to-a-cluster
I think so

---

Maybe something like:

```suggestion
  channel: pipelines-1.8
  startingCSV: openshift-pipelines-operator-rh.v1.8.0
  installPlanApproval: Automatic
```